uvicorn/protocols/http/httptools_impl.py

```diff
@@ -74,13 +74,16 @@ def __init__(self, scope):
 
 
 class HttpToolsProtocol(asyncio.Protocol):
-    def __init__(self, config, server_state):
+    def __init__(self, config, server_state, _loop=None):
         if not config.loaded:
             config.load()
 
+        if config.loop_instance is None:
+            config.setup_event_loop()
```

Comment:
I guess we can drop these.

Reply:
Do you mean the `loop_instance` reference or the call itself ?

Reply:
I have removed the reference

Reply:
Oh sorry, you mean the call in the protocols. Yes, I think you are right: the loop will always be already initialized at this point.